cogs/tickets.py

```diff
@@ -190,6 +190,12 @@ async def close(self, ctx, *, reason: str = None):
         await log_channel.send(embed=embed,
                                file=discord.File(f"transcripts/{ctx.channel.name}.html", "transcript.html"))
 
+    @commands.command(hidden=True, aliases=['sc', 'staffclaim'])
+    @staff_or_trainee
+    async def staff_claim(self, ctx, arg):
+        restrict_ticket_command_usage(ctx)
+        await ctx.channel.name.id.edit(name=f"{arg}{ctx.channel.name}")
```

Comment:
This is not how you change a channel name, `await ctx.channel.edit(name=f"name")` would work correctly. See [here](https://discordpy.readthedocs.io/en/latest/api.html#discord.TextChannel.edit) for more information.